coredb-cli/src/commands/create.rs

```diff
@@ -0,0 +1,18 @@
+use super::{ResourceType, SubCommand};
```

Comment:
I _think_ alternatively, you could add a `lib.rs`. Then in `lib.rs`, `pub mod everything.rs`. Then through the project, including `main.rs` you can reference all the functions and structs like `coredb-cli::commands::{get, create}`

Reply:
Ok I will try this